Client/src/Pages/Incidents/index.jsx

```diff
@@ -57,7 +57,7 @@ export default function CustomizedTables() {
         </TableHead>
         <TableBody>
           {rows.map((row, index) => (
-            <StyledTableRow key={row.name}>
+            <StyledTableRow key={index}>
```

Comment:
I think this is still a WIP while we wait for data to be passed in, but index should not be used for keys.  Since the order of the items isn't necessarily fixed the key is unstable, so we really should use a unique and stable key.  We can change this when the component is ready for use

Reply:
Sounds good thank you for your review!